plugins/chrome-ux-report/src/api.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createApiRef, DiscoveryApi } from '@backstage/core';
+import { Config } from '@backstage/config';
+import { getPeriod } from './utils';
+
+export const chromeuxReportApiRef = createApiRef<ChromeUXReportApi>({
+  id: 'plugin.chromeuxreport.service',
+  description: 'Used to make requests towards ChromeUXReport API',
+});
+
+export type ChromeUXReport = {
+  getChromeUXMetrics(origin: string): Promise<string>;
+};
+
+/**
+ * API to talk to chrome-ux-report-backend.
+ *
+ * @property {string} apiOrigin Set to chromeuxreport.requestUrl as the URL for chrome-ux-report-backend API
+ */
+export class ChromeUXReportApi implements ChromeUXReport {
```

Comment:
Flip these around, `class ChromeUXReportClient implements ChromeUXReportApi`, rename the interface to `ChromeUXReportApi` and keep using it as the type of the `ApiRef`.